components/resourceDetails/resourceSharing/agentAccessTable/__snapshots__/index.test.jsx.snap

```diff
@@ -252,7 +198,7 @@ exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
 </DocumentFragment>
 `;
 
-exports[`AgentAccessTable with consent based agents renders a list of permissions including consent based agent 1`] = `
+exports[`AgentAccessTable with agents renders a list of permissions 1`] = `
```

Comment:
`AgentAccessTable with access grants based agents renders a list of permissions including access grants  based agent `

Reply:
That's the name of the test. You need to change the title in the test and then update the snapshot (I might have missed this in the actual test file)